### ISSUE TITLE
Rectangular Solid designs cause server failure when opened

### ISSUE THREAD

**Issue by grumpyinca** [thegrumpys/odop#261]
So far, I have identified two Rectangular Solid designs in the development database (ViolTest and USPS_MachinableDimensions) will cause the server to fail when attempting File : Open.
The server must be restarted before successful operation can be resumed.

See Slack (2/6/2019) for details of error messages.

**Comment by 1fifoto** [comment]
GrumpyInCA [5:49 PM]
    Hi.  This is not urgent.  I seem to have created and saved a Rectangular Solid design that causes indigestion for the server when attempting to open it.  I am operating in the master branch.  I have not merged branch 180.  The name of the design is ViolTest.  The error message in the server window is: (edited) 
    SERVER: In GET /api/v1/designtypes/Solid/designs/ViolTest
    C:\Users\Mike\git\odop\node_modules\mysql\lib\protocol\Parser.js:80
           throw err; // Rethrow non-MySQL errors
           ^
    
    SyntaxError: Unexpected token
    in JSON at position 3236
       at Object.parse (native)
       at Query.<anonymous> (C:\Users\Mike\git\odop\server.js:108:26)
       at Query.<anonymous> (C:\Users\Mike\git\odop\node_modules\mysql\lib\Connection.js:502:10)
       at Query._callback (C:\Users\Mike\git\odop\node_modules\mysql\lib\Connection.js:468:16)
       at Query.Sequence.end (C:\Users\Mike\git\odop\node_modules\mysql\lib\protocol\sequences\Sequence.js:83:24)
       at Query._handleFinalResultPacket (C:\Users\Mike\git\odop\node_modules\mysql\lib\protocol\sequences\Query.js:139:8)
       at Query.EofPacket (C:\Users\Mike\git\odop\node_modules\mysql\lib\protocol\sequences\Query.js:123:8)
       at Protocol._parsePacket (C:\Users\Mike\git\odop\node_modules\mysql\lib\protocol\Protocol.js:278:23)
       at Parser.write (C:\Users\Mike\git\odop\node_modules\mysql\lib\protocol\Parser.js:76:12)
       at Protocol.write (C:\Users\Mike\git\odop\node_modules\mysql\lib\protocol\Protocol.js:38:16)
    [nodemon] app crashed - waiting for file changes before starting...
    
    The error message in the console is:
    FileOpen.js:47 GET http://localhost:3000/api/v1/designtypes/Solid/designs/ViolTest 500 (Internal Server Error)
    
    Bri [5:51 PM]
    Interesting
    
    GrumpyInCA [5:51 PM]
    The error message in the client window is:
    Proxy error: Could not proxy request /api/v1/designtypes/Solid/designs/ViolTest from localhost:3000 to http://localhost:5000/.
    See https://nodejs.org/api/errors.html#errors_common_system_errors for more information (ECONNRESET).
    
    Bri [5:52 PM]
    Things to deal with this weekend
    
    GrumpyInCA [5:54 PM]
    OK.  I will open an issue.   F.Y.I.  My Fiber-To-The-Home installation is scheduled for tomorrow morning.  This could end up being a significant distraction.

**Comment by 1fifoto** [comment]
Modified to server.js to replace all single quotes with double quotes for MySQL

    value = value.replace(/[']/ig,"''"); // replace one single quote with an two single quotes throughout

Eliminate all double quotes, backslash and forward slash characters from property input in FileProperties

    value: value.replace(/["\\/]/ig, '') // replace invalid JSON characters with nothing throughout

Merged into master, closing
